internal/utils/paths.go

```diff
@@ -41,6 +46,13 @@ func DBDir(home string, name string) string {
 	return filepath.Join(home, "db", fmt.Sprintf("%s.sqlite3", name))
 }
 
+func FileExists(path string) bool {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return false
+	}
+	return true
```

Comment:
```suggestion
	stat, err := os.Stat(path)
	return err == nil && stat.Mode().IsRegular()
```

I think we should treat the file as missing if there is _any_ error. For example, if the directory is not readable, we want to return false here. Also, we want to make sure the target path is a regular file.